reader-osm/src/test/java/com/graphhopper/GraphHopperIT.java

```diff
@@ -72,28 +75,29 @@ public static void beforeClass() {
                 importOrLoad();
     }
 
-    @AfterClass
+    @AfterAll
     public static void afterClass() {
         Helper.removeDir(new File(graphFileFoot));
     }
 
-    @Before
+    @BeforeEach
     public void setUp() {
         Helper.removeDir(new File(tmpGraphFile));
     }
 
-    @After
+    @AfterEach
     public void tearDown() {
         Helper.removeDir(new File(tmpGraphFile));
     }
 
-    @Test
-    public void testMonacoWithInstructions() {
+    @ParameterizedTest
+    @CsvSource({DIJKSTRA + ",1110", DIJKSTRA_BI + ",892", ASTAR + ",699", ASTAR_BI + ",396"})
```

Comment:
This is quite nice, especially for testing with different api parameters. Each element of the array is split into separate parameters to the test method (`algo` and `expectedVisitedNodes`) and there are default converters (e.g. expectedVisitedNodes is parsed to an int here. So with these two annotations I was able to run the test I was talking about in the initial comment for different algorithms.

Reply:
Its also nice to see the visited nodes counts for the different algos here :)

Reply:
Interesting! Looks good.